service/src/main/java/com/ge/predix/acs/attribute/connectors/ZoneConfiguration.java

```diff
@@ -0,0 +1,29 @@
+    package com.ge.predix.acs.attribute.connectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ZoneConfiguration {
+    @Autowired
+    private DefaultResourceAttributeReader defaultResourceAttributeReader;
+    @Autowired
+    private DefaultSubjectAttributeReader defaultSubjectAttributeReader;
+
+    private AttributeReader resourceAttributeReader;
```

Comment:
The idea is to construct resourceAttributeReader based on zone configuration read from the database.